tests/test_unclassified_job.py

```diff
@@ -26,3 +25,13 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    assert len(page.result_sets[0].testfailed_job_title) >= 0
```

Comment:
> yes it will fail if there aren't any test failures

You could perhaps not limit the test to just the first result set, which would increase the chances of there being testfailed jobs.

> I'm unclear what you mean about using the jobs property in this way

I mean something like `page.result_sets[0].jobs[0].title` rather than creating a new property for a certain job status on the page object. If you used this you should even be able to iterate or compile a list of all jobs displayed and ensure that they are unclassified. From your asserts I gather than a job with a status of testfailed means the same as unclassified? Otherwise I'm not sure how we're checking the filter is working as expected.